/ebs_supervisor_node:
  ros__parameters:
    generic:
      WCET: 5000000 # ms
      period: 10000000 # 10ms
      deadline: 10000000
    topic:
      canTxTopic: "/canbus/tx/msg"
      ebsStatusTopic: "/ebs/status"